core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -138,6 +139,36 @@ public void testNullStatusOnKraftCommandAlterClientMetrics() {
         assertEquals("Completed updating config for client-metric cm.", message);
     }
 
+    @ClusterTest
+    public void testAddConfigKeyValuesUsingCommand() throws Exception {
+
+        try (Admin client = cluster.admin()) {
+            NewTopic newTopic = new NewTopic("test-topic", 1, (short) 1);
```

Comment:
Nit:  `String topic = "test-topic";` and replace all "test-topic" with `topic`.

Reply:
Okay.

> Nit: `String topic = "test-topic";` and replace all "test-topic" with `topic`.

Reply:
My meaning is using variable to replace constant but this PR is good enough.